actions/form/class.Instance.php

```diff
@@ -227,4 +230,42 @@ private function isEmptyLabel($element): bool
         return $element instanceof tao_helpers_form_elements_Label
             && empty($element->getRawValue());
     }
+
+    /**
+     * Check if the Solar design is enabled and the postfix has not yet been added
+     *
+     */
+    private function isContainPostfix(string $language): bool
+    {
+        $pattern = '/' . self::LANG_POSTFIX . '$/';
+
+
+        return !Layout::isSolarDesignEnabled() || preg_match($pattern, $language, $matches);
```

Comment:
```suggestion
        return !Layout::isSolarDesignEnabled() || preg_match($pattern, $language);
```
We don't need $matches as we don't use it

Reply:
Also I think that `preg_match($pattern, $language)` return integers/bool not bool only and it can fail as you function has bool return type only. Probably we need to change it to `preg_match($pattern, $language) === 1`